mobile_packet_verifier/src/accumulate.rs

```diff
@@ -41,8 +41,8 @@ pub async fn accumulate_sessions(
         }
         sqlx::query(
             r#"
-            INSERT INTO data_transfer_sessions (pub_key, payer, uploaded_bytes, downloaded_bytes, first_timestamp, last_timestamp)
-            VALUES ($1, $2, $3, $4, $5, $5)
+            INSERT INTO data_transfer_sessions (pub_key, payer, uploaded_bytes, downloaded_bytes, subscriber_id, first_timestamp, last_timestamp)
```

Comment:
Why do we have a `first_timestamp` and `last_timestamp` if they are always set to the same value?

Reply:
a typo?

Reply:
yup missed it changing in the on conflict.